app/views/admin/configs/show.html.erb

```diff
@@ -234,11 +228,52 @@
                                              placeholder: Constants::SiteConfig::DETAILS[:recaptcha_secret_key][:placeholder] %>
                           </div>
                         </div>
+                        <div class="mt-6">
+                          <% if SiteConfig.allow_both_email_signup_and_login %>
+                            <button
+                              class="crayons-btn crayons-btn--danger <%= cannot_disable_email_auth ? "crayons-tooltip" : "" %>"
```

Comment:
I think this could be a helper in and of itself:
```ruby
def disable_email_tooltip_class
    (SiteConfig.invite_only_mode || authentication_enabled_providers.none?) ? "crayons-tooltip" : ""
end
```

And then you could could just call the helper here and not even have to do any conditional inline checks
```suggestion
                              class="crayons-btn crayons-btn--danger <%= disable_email_tooltip_class %>"
```

Since you're going to the trouble of making a helper, may as well make it do more work for you 😉